spec/models/peer_review_spec.rb

```diff
@@ -89,7 +89,7 @@
     end
   end
 
-  describe '#has_marks_or_annotations?' do
+  describe 'has_marks_or_annotations?' do
```

Comment:
`self.get_num_marked` and `self.get_num_collected` are marked with `self.`, which from my knowledge, makes them class methods rather than instance methods. However, their corresponding test cases are also prefixed with `#`. Which naming convention should these test cases follow?